Header header
float32 x
float32 y
float32 z
int32 frame_seq
int32 object_id
bool is_human
